### DISCUSSION TITLE
Count days between 2 dates

### DISCUSSION THREAD

**Discussion by LMRig** [moment/luxon#1184] [Q&A]
Hello, please, is there any way how to count days?
```
date1 = "18.04.2022";
date2 = DateTime.now().toFormat('dd.MM.yyyy') //Output is 16.04.2022
```

I need to count how many days are between these 2.
Does anyone know?
Or any working solution on how to compare 2 days and get a number of days between them?

Thank you.

**Comment by icambron** [comment]
use `diff()`:

```js
const date1 = DateTime.fromISO("2022-04-23")
const date2 = DateTime.now().startOf("day") // it's the 21st today
date1.diff(date2, "days").days //=>  2 
```
